vulcanai2/models/dnn.py

```diff
@@ -114,12 +114,13 @@ def __init__(self, name, config, in_dim=None, save_path=None,
                  activation=nn.ReLU(), pred_activation=None,
                  optim_spec={'name': 'Adam', 'lr': 0.001},
                  lr_scheduler=None, early_stopping=None,
-                 criter_spec=nn.CrossEntropyLoss()):
+                 criter_spec=nn.CrossEntropyLoss(),
+                 device="cpu"):
```

Comment:
same: let's make sure the defaults of `cuda:0` are consistent.

Reply:
👍